### ISSUE TITLE
Import Each Less File Only Once

### ISSUE THREAD

**Issue by michael-savory** [less/less.js#212]
This is somewhat a duplicate of issue #49, however I did not find any clarification on what the best strategy was.  Also, I should mention that I'm doing all of my compiling on the server side using node.

Here is my problem.

home.less:

@import "config";

@import "reset";

@import "core";

@import "base";

How I would like this to work is that it's like I'm specifying all of my dependencies at the top of my less file, and it will get compiled down into one nice css file.

However, what you get instead is:

-- config.css

-- reset.css

-- config.css

-- core.css

-- config.css 

-- core.css

-- base.css

-- home.css

Basically, everyone depends on "config", because that's where all my colors and functions are located, and it would be nice if that was only loaded only once.  Notice, in my specific case "core" gets loaded twice as well.

Is there a workaround, a hack, an alternative, or a fork that deals with this scenario.  I feel like it's valuable from the standpoint that you could have css files like the following:

home.css

index.css

posts.css

blogs.css

And these would more or less dynamically create themselves with only the necessary css components based upon the imports you specify.

Cheers.
